mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -320,5 +339,88 @@ def _download_file(self, remote_file_path, local_path):
         read_credentials = self._get_read_credentials(self.run_id, run_relative_remote_file_path)
         self._download_from_cloud(read_credentials.credentials, local_path)
 
+    def download_artifacts(self, artifact_path, dst_path=None):
+        """
+        Parallelized implementation of `download_artifacts` for Databricks.
+        """
+
+        def download_file(fullpath):
+            """
+            Submit a download of `fullpath` to the thread pool and return the
+            resultant Future.
+            """
+            fullpath = fullpath.rstrip("/")
```

Comment:
How about changing this to
full_path = os.path.abspath(full_path)?
It handles more cases than ending '/'.